qiskit/assembler/disassemble.py

```diff
@@ -13,13 +13,16 @@
 # that they have been altered from the originals.
 
 """Disassemble function for a qobj into a list of circuits and its config"""
-
+from typing import Any, Dict, List, NewType, Tuple, Union
```

Comment:
NewType is cool! Has this always been around? Can't believe I didn't know it.... 😄